pax/plugins/signal_processing/PulseShapeDiscrimination.py

```diff
@@ -0,0 +1,146 @@
+                    if((leftbound != 0) and (rightbound!= 0)):
+                        f160_int += sum(itertools.islice(signalpe, leftbound, min(rightbound, len(signalpe) - 1)))
+                    leftbound, rightbound = self.GetIntegrationBounds(int(s1.center_time / 10.0 - len(s1.sum_waveform) / 2.0) + starttime, s1right, pulse.left, 17)
+                    if((leftbound != 0) and (rightbound!= 0)):
+                        f170_int += sum(itertools.islice(signalpe, leftbound, min(rightbound, len(signalpe) - 1)))
+                    leftbound, rightbound = self.GetIntegrationBounds(int(s1.center_time / 10.0 - len(s1.sum_waveform) / 2.0) + starttime, s1right, pulse.left, 18)
+                    if((leftbound != 0) and (rightbound!= 0)):
+                        f180_int += sum(itertools.islice(signalpe, leftbound, min(rightbound, len(signalpe) - 1)))
+                    leftbound, rightbound = self.GetIntegrationBounds(int(s1.center_time / 10.0 - len(s1.sum_waveform) / 2.0) + starttime, s1right, pulse.left, 19)
+                    if((leftbound != 0) and (rightbound!= 0)):
+                        f190_int += sum(itertools.islice(signalpe, leftbound, min(rightbound, len(signalpe) - 1)))
+                    leftbound, rightbound = self.GetIntegrationBounds(int(s1.center_time / 10.0 - len(s1.sum_waveform) / 2.0) + starttime, s1right, pulse.left, 20)
+                    if((leftbound != 0) and (rightbound!= 0)):
+                        f200_int += sum(itertools.islice(signalpe, leftbound, min(rightbound, len(signalpe) - 1)))
+                    leftbound, rightbound = self.GetIntegrationBounds(int(s1.center_time / 10.0 - len(s1.sum_waveform) / 2.0) + starttime, s1right, pulse.left, 50)
+                    if((leftbound != 0) and (rightbound!= 0)):
+                        total_int += sum(itertools.islice(signalpe, leftbound, min(rightbound, len(signalpe) - 1)))
+            if(total_int == 0):
+                continue
+            s1.nick = int(s1.center_time / 10.0 - len(s1.sum_waveform) / 2.0) + starttime
+            s1.f10 = f10_int/total_int
+            s1.f20 = f20_int/total_int
+            s1.f30 = f30_int/total_int
+            s1.f40 = f40_int/total_int
+            s1.f50 = f50_int/total_int
+            s1.f60 = f60_int/total_int
+            s1.f70 = f70_int/total_int
+            s1.f80 = f80_int/total_int
+            s1.f90 = f90_int/total_int
+            print(s1.f90)
+            s1.f100 = f100_int/total_int
+            s1.f110 = f110_int/total_int
+       	    s1.f120 = f120_int/total_int
+       	    s1.f130 = f130_int/total_int
+       	    s1.f140 = f140_int/total_int
+       	    s1.f150 = f150_int/total_int
+       	    s1.f160 = f160_int/total_int
+       	    s1.f170 = f170_int/total_int
+       	    s1.f180 = f180_int/total_int
+       	    s1.f190 = f190_int/total_int
```

Comment:
__[flake8]__
* [E101] indentation contains mixed spaces and tabs
* [W191] indentation contains tabs

<!-- Comment by [SideCI](https://sideci.com) -->